source/frontend/src/features/properties/map/acquisition/detail/checklist/AcquisitionChecklistView.tsx

```diff
@@ -72,27 +76,33 @@ export const AcquisitionChecklistView: React.FC<IAcquisitionChecklistViewProps>
                 labelWidth="6"
                 noGutters
               >
-                <StyledChecklistItem>
-                  <StyledChecklistItemAudit>
-                    {!isDefaultState(checklistItem) && (
-                      <>
-                        <UserNameTooltip
-                          userName={checklistItem.appLastUpdateUserid}
-                          userGuid={checklistItem.appLastUpdateUserGuid}
-                        />
-                        <em> {prettyFormatDate(checklistItem.appLastUpdateTimestamp)}</em>
-                      </>
-                    )}
-                  </StyledChecklistItemAudit>
-                  <StyledChecklistItemStatus
-                    color={mapStatusToColor(checklistItem.statusTypeCode?.id)}
-                  >
-                    {checklistItem.statusTypeCode?.description}
-                  </StyledChecklistItemStatus>
-                  <StyledChecklistItemIcon>
-                    <StatusIcon status={checklistItem.statusTypeCode?.id} />
-                  </StyledChecklistItemIcon>
-                </StyledChecklistItem>
+                <Row className="no-gutters">
```

Comment:
nit: in the future can we try and keep enhancement PRs like this separate? I don't want to whine too much because I like the improvements, but checklists don't really have much to do with agreements.

Reply:
Yeah I agree. I was centralizing the styles and refactored some of the checklist code